include/jinja2cpp/filesystem_handler.h

```diff
@@ -2,6 +2,7 @@
 #define JINJA2CPP_FILESYSTEM_HANDLER_H
 
 #include <nonstd/variant.hpp>
+#include <nonstd/optional.hpp>
 
 #include <iostream>
```

Comment:
is iostream needed here?

Reply:
Probably yes, for the declaration of basic_i/ostream etc.